fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -121,7 +125,14 @@ public Database(long id, String name) {
     }
 
     public void readLock() {
+        long startMs = System.nanoTime() / 1000000;
```

Comment:
use tryLock same like Catalog.tryLock may be better?

Reply:
This PR only add some log, it did not change existing code logic. There are too many places call readLock & writeLock, we cannot change them all